portfolio.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Beryl's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header class="nav_section header">
+      <div class="my_logo">
+        <h2 class="logo"><a class="logo" href="#">GAKII </a></h2>
+      </div>
+      <nav>
+        <div class="menu">
+          <img src="./images/Union.png" class="log" />
+        </div>
+        <ul class="links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a class="fa-regular fa-envelope"></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline1">
+      <article>
+        <h1>I’m Gakii<br />Glad to see,You!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work
+          and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+      </article>
+        <div class="social-media">
+          <ul>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/Linkedin%20icon.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+          </ul>
+        </div>
+    </section>
+    <section class="grid_container">
+      <div class="block1">
+        <div class="block1a">
+          <img src="images/Snapshoot Portfolio1.png" />
+        </div>
+          <div class="item1b">
+            <h2 class="tonic">Tonic</h2>
+            <div class="frame2 tonic">
+              <span>CANOPY</span>
+              <ul class="role">
+                <li>Back End Dev</li>
+                <li>2015</li>
```

Comment:
- [ ] Good work on making the changes but there are some aspects you can improve on
 - Some elements appear to be out of position, Kindy position them correctly
 
![image](https://user-images.githubusercontent.com/98453979/229179676-f6b22121-991f-4ec5-81a9-2a671ddfbbaf.png)

 - Increase the padding around the languages to match the original design

![image](https://user-images.githubusercontent.com/98453979/229179894-039853fc-0f61-4fe5-b157-727bdb6a33a9.png)

 - In your `languages` section, position your elements correctly, to match the original design.

![image](https://user-images.githubusercontent.com/98453979/229180234-53cc45a1-cf44-4a3c-a221-ed0321a8f0be.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/229180358-2918b418-0deb-4393-9d68-05025dde28c7.png)